proto/tsopb.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package tsopb;
```

Comment:
According to our last sync, my understanding is that we'll provide these APIs as storage interface used by tso service internal.